rrr_arm:
  # Publish all joint states
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers
  joint1_position_controller:
    type: position_controllers/JointPositionController
    #type: effort_controllers/JointPositionController
    joint: joint_1
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller:
    type: position_controllers/JointPositionController
    joint: joint_2
  joint3_position_controller:
    type: position_controllers/JointPositionController
    joint: joint_3   
  joint_motor_controller:
    type: velocity_controllers/JointGroupVelocityController
    joints: 
        - joint_front_right_prop
        - joint_front_left_prop
        - joint_back_left_prop
        - joint_back_right_prop
  #  gains:
   #     joint_front_right_prop: {p: 10000, i: 1, d: 1000}
   #     joint_front_left_prop: {p: 10000, i: 1, d: 1000}
   #     joint_back_left_prop: {p: 10000, i: 1, d: 1000}
   #     joint_back_right_prop: {p: 10000, i: 1, d: 1000}
  # PID Controllers params
  # gazebo_ros_control:
  #     pid_gains:
  #       joint_1:
  #         p: 100.0
  #         i: 0.01
  #         d: 10.0
  #       joint_2:
  #         p: 100.0
  #         i: 0.01
  #         d: 10.0
  #       joint_3:
  #         p: 100.0
  #         i: 0.01
  #         d: 10.0
  #       joint_4:
  #         p: 100.0
  #         i: 0.01
  #         d: 10.0
  #       gripper_joint_1:
  #         p: 100.0
  #         i: 0.01
  #         d: 10.0
  #       gripper_joint_2:
  #         p: 100.0
  #         i: 0.01
  #         d: 10.0